src/schema/rules/nifti_header.yaml

```
@@ -0,0 +1,40 @@
+---
+- nifti_header:
+  dim_info:
+    freq:
+      type: int
+    phase:
+      type: int
+    slice:
+      type: int
```

Maybe helpful hints for implementors; should not try to make these parseable.

```suggestion
    freq:
      type: int
      # value = dim_info & 0x03
    phase:
      type: int
      # value = (dim_info >> 2) & 0x03
    slice:
      type: int
      # value = (dim_info >> 4) & 0x03
```